interface-elements-for-web/articles/menu/menu-scrolling/menu-scrolling.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Menu Scrolling
+author: Svetlana Nikulina
+legacyId: 9383
+---
+# Menu Scrolling
+A submenu displays scroll buttons if it does not fit into the visible document area. To scroll through a submenu use the button.
```

Comment:
"To scroll through a submenu use the button." Which button?

Reply:
Updated.